### DISCUSSION TITLE
What's up tonight: filter for objects above the landscape

### DISCUSSION THREAD

**Discussion by markusd112** [Stellarium/stellarium#2311] [Q&A] [answered]
Hi,
is there any possibility to filter the "What's up tonight" object list for objects that are visible and above the polygonal landscape line? It is a little bit tricky to find some objects because you have to check manually each object if it is really visible from my location or not... 
Thanks
Markus

**Comment by alex-w** [comment, marked as answer]
We have plans for implementing described behaviour for all types of landscapes, but this is in our long-term TODO list. Of course, pull requests are welcome.
